modules/renter/uploadheap.go

```diff
@@ -236,6 +238,30 @@ func (r *Renter) managedBuildChunkHeap(hosts map[string]struct{}) {
 	// the heap.
 	id := r.mu.Lock()
 	for _, file := range r.files {
+		// check for local file
+		tf, exists := r.persist.Tracking[file.name]
+		if exists {
```

Comment:
If the path doesn't exist for some reason, we still want to check if the redundancy is below 1x and log a warning if that's not the case. 
@lukechampine @DavidVorick is it even possible that a file doesn't have an entry in the tracking map? maybe we should also add a sanity check and logging for that case.

Reply:
it used to be, and it may be again in the future: if you loaded a shared .sia file, you wouldn't have a local file to track.

Reply:
So should I change anything or leave as is?